_posts/en/newsletters/2023-03-15-newsletter.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Bitcoin Optech Newsletter #242'
+permalink: /en/newsletters/2023/03/15/
+name: 2023-03-15-newsletter
+slug: 2023-03-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Service bit for Utreexo:** Calvin Kim [posted][kim utreexo] to the
+  Bitcoin-Dev mailing list that software currently designed for
+  experimentation on signet and testnet will be using P2P protocol
+  service bit 24.  The experimental software provides support for
+  [Utreexo][topic utreexo], a protocol for allowing full verification of
+  transaction by software which doesn't store a copy of the UTXO set,
+  saving up to 5 GB of disk space compared to a modern Bitcoin Core full
+  node without any reduction in security.  A Utreexo node needs to
+  receive extra data when it receives a transaction (or a block full of
+  transactions), so the service bit will help a node find peers capable
+  of providing the extra data.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.16.0-beta.rc2][] is a release candidate for a new major
+  version of this popular LN implementation.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25740][] allows a node using [assumeUTXO][topic
+  assumeutxo] to download all blocks before the assumed-valid block,
```

Comment:
Maybe make it a bit clearer that this is a step towards AssumeUTXO, but it's not ready for use yet.  While this PR added important validation logic for assumeUTXO, at least the p2p part to actually download the blocks processed by validation (PR 24008), plus the RPC changes to let a user load a chainstate would still need to be merged for that.

Reply:
In noting that this is part of the assumeutxo project, I think it would be nice to also work in this link for context https://github.com/bitcoin/bitcoin/projects/11

Reply:
Yeah, I first thought that this was supposed to be the “assume-UTXO” block and a typo, but at second glance, the “assumed-valid” block makes sense. Perhaps the connection of the two should be substantiated.